store/mongo/datastore_mongo_test.go

```diff
@@ -653,7 +650,7 @@ func TestSetSessionRecording(t *testing.T) {
 					Keys: bson.D{{Key: "expire_ts", Value: 1}},
 					Options: mopts.Index().
 						SetBackground(true).
-						SetExpireAfterSeconds(0).
+						SetExpireAfterSeconds(int32(tc.Expiration.Seconds())).
```

Comment:
good joke: this test never worked. apparently mongo check the expiration with a grain of 1 minute. the test passed because there was a bug, we set [Tenant: "000000000000000000000002",](https://github.com/mendersoftware/deviceconnect/blob/master/store/mongo/datastore_mongo_test.go#L626) in the test case input, but the tests are run with ["000000000000000000000001", DbName,](https://github.com/mendersoftware/deviceconnect/blob/master/store/mongo/datastore_mongo_test.go#L646) -- so the record was never actually found, which was taken as expired and removed document scenario. The move to single db exposed that. There is no way I know of atm to make mongo scan more often. there are two possibilities, we remove the test, or we keep it as it is now sleeping at least 60s.

Reply:
I'm okay with the 60 seconds sleep!

Reply:
thank you!